NPE in FileUploader

Google dev console reports this 139x, so it would be quite nice to have this fixed.
It seems as somehow intent is null?

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:2719)
  at android.app.ActivityThread.access$2100 (ActivityThread.java:135)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1293)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:136)
  at android.app.ActivityThread.main (ActivityThread.java:5017)
  at java.lang.reflect.Method.invokeNative (Native Method)
  at java.lang.reflect.Method.invoke (Method.java:515)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:798)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:614)
  at dalvik.system.NativeStart.main (Native Method)
Caused by: java.lang.NullPointerException: 
  at com.owncloud.android.files.services.FileUploader.onStartCommand (FileUploader.java:503)
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:2702)
 ```
which is:

https://github.com/nextcloud/android/blob/45bcdebd9dd51c60ce7870dbe0547d2243cfe54c/src/main/java/com/owncloud/android/files/services/FileUploader.java#L503-L503

--- a/src/main/java/com/owncloud/android/files/services/FileUploader.java
+++ b/src/main/java/com/owncloud/android/files/services/FileUploader.java
@@ -499,6 +499,11 @@ public class FileUploader extends Service
         Log_OC.d(TAG, "Starting command with id " + startId);
 
         startForeground(FOREGROUND_SERVICE_ID, mNotification);
+        
+        if (intent == null) {
+            Log_OC.e(TAG, "Intent is null");
+            return Service.START_NOT_STICKY;
+        }
 
         boolean retry = intent.getBooleanExtra(KEY_RETRY, false);
         AbstractList<String> requestedUploads = new Vector<String>();

